docs/general/performance.md

```diff
@@ -82,3 +103,32 @@ new Chart(ctx, {
     }
 });
 ```
+
+### Disable Point Drawing
```

Comment:
I noticed that this logic is in the `point.draw`. It might be worth moving it to the line controller because there might be some logic we could skip in that case. E.g. we probably don't need to call `point.pivot` for every point if we're not going to be drawing them

Reply:
Line is drawn based on points, so we'll need to pivot them even if radius is 0.